### PR TITLE
added readme file

### PR DESCRIPTION
**Pull request by Den-moroz**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+   If you prefer to run the Book Store API on your local machine, follow these steps:
+
+   ### Prerequisites
+
+   Before you begin, ensure you have the following prerequisites installed on your system:
+
+    - [Java 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)
+    - [MySQL database](https://www.mysql.com/downloads/)
+    - [Docker](https://docs.docker.com/get-docker/)
```
Comment (vovasalyha) [inline Readme.md:94]: I think that you need to add maven to prerequisites too as it is used in the installation process

**Review by vovasalyha — APPROVED** [review]
GJ

File: Readme.md (hunk 1 above)
Comment (boroda4436) [reply Readme.md:94]: `mvnv` could be used as an alternative for installing Maven locally

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+   You've now successfully set up and launched the Book Store API locally.
+
+## Postman Collection
+For your convenience, i've created a Postman collection that includes sample requests for various API endpoints. You can download it [here](BookStore-api.postman_collection.json) and import it into your Postman workspace to get started quickly.
+
+## Challenges Faced
+During the development of this project, i encountered a few challenges, such as data modeling complexities and security implementations. However, through collaboration and extensive testing, i successfully overcame these challenges to deliver a reliable and secure API.
+
+## Conclusion
```
Comment (boroda4436) [inline Readme.md:151]: Chat GPT?

File: Readme.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+- **ShoppingCartController**: Manages shopping cart information, allowing users to create, retrieve, update, and delete cart items.
+
+- **OrderController**: Handles order management, including creating orders, updating order status, and retrieving order history.
+
+# Installation and Usage
+
+You can interact with the Book Store API in two ways:
+
+1. **Using Swagger (No Installation Required)**
```
Comment (boroda4436) [inline Readme.md:66]: It's better to mention that this example is deployed to AWS, that's why No installation required

**Review by boroda4436 — COMMENTED** [review]
As an idea: you can write a loom video of how your app works and add it to Readme.

File: Readme.md (hunk 1 above)
Comment (vovasalyha) [reply Readme.md:94]: Then it's better to specify it in the installation guide, because now there is a `mvn` command used, not a wrapper

**Pull request merged**